egs/uw3/v1/local/nnet3/compare.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Please update this to the same script in Ashish's PR. I asked him to fix it, you don't need to do it again.